agents/mysql/perfschema/perfschema_test.go

```diff
@@ -453,4 +453,19 @@ func TestPerfSchema(t *testing.T) {
 		require.NoError(t, err)
 		tests.LogTable(t, structs)
 	})
+
+	t.Run("DisableQueryExamples", func(t *testing.T) {
+		m := setup(t, db, true)
+		_, err = db.Exec("SELECT 1, 2, 3, 4, id FROM city WHERE id = 1")
+		require.NoError(t, err)
+
+		require.NoError(t, m.refreshHistoryCache())
+
+		buckets, err := m.getNewBuckets(time.Date(2019, 4, 1, 10, 59, 0, 0, time.UTC), 60)
+		require.NoError(t, err)
+
+		for _, b := range buckets {
+			assert.Empty(t, b.Common.Example)
```

Comment:
I think it will not hurt to check that only example is empty, but something else is present

Reply:
^

Reply:
I see that @percona-csalguero already added a few lines above to check fingerprint. What else should be checked?

Reply:
I missed that :)